matplotcheck/base.py

```diff
@@ -225,11 +249,13 @@ def assert_axis_off(self, m="Axis lines are displayed on plot"):
 
         Parameters
         ----------
-        m: string error message if assertion is not met
+        m : string
```

Comment:
ok since you are editing this. why don't we give this a more expressive name. I think it should be message rather than m at a minimum. what do you think @ryla5068 ? we normally try to use expressive variable and parameter names to make the code easier to read so at a minimum i'd like m to be message.

Reply:
I will address this in another PR and roll this into issue #106